pkgs/development/tools/build-managers/remake/default.nix

```diff
@@ -1,27 +1,68 @@
-{ lib, stdenv, fetchurl, readline }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchftpdir
+, autoreconfHook
+, gettext
+, perl
+, pkg-config
+, readline
+, texinfo
+}:
 
 stdenv.mkDerivation rec {
   pname = "remake";
-  remakeVersion = "4.1";
-  dbgVersion = "1.1";
+  remakeVersion = "4.3";
+  dbgVersion = "1.5";
   version = "${remakeVersion}+dbg-${dbgVersion}";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/project/bashdb/remake/${version}/remake-${remakeVersion}+dbg${dbgVersion}.tar.bz2";
-    sha256 = "1zi16pl7sqn1aa8b7zqm9qnd9vjqyfywqm8s6iap4clf86l7kss2";
+  src = fetchFromGitHub {
+    owner = "rocky";
+    repo = "remake";
+    rev = "release_${remakeVersion}%2Bdbg-${dbgVersion}";
+    sha256 = "0fidz2jpy2c9pn16xv485jv575kikhs0dlcdh1qfl5l8xv1px79a";
+  };
+  po_files = fetchftpdir {
+    url = "http://translationproject.org/latest/make";
+    sha256 = "0y8sw9isjmbb8466lh14sk70zl49nyql2nf6494ln6083jzsgcil";
```

Comment:
`http://translationproject.org/latest/make` seems like a very mutable URL, and will surely soon break. Is there no better URL?

Reply:
I sent an email to comments@translationproject.org, asking about permalinks. (I also looked a bit on the site myself, but couldn't find any permalinks.)

Personally, this is not a hard blocker, but would be nice to avoid using such mutable URLs if possible.

Reply:
https://github.com/rocky/remake/issues/46 mentions to use [released sources](https://github.com/rocky/remake/issues/46#issuecomment-532922782) instead of using github, to avoid requiring internet access during the build (a.k.a. downloading unpinned translation files).

@SCOTT-HAMILTON : Your thoughts?

Reply:
I've opened a discourse topic, I'll draft this PR until [here](https://discourse.nixos.org/t/autoreconf-vs-already-configured-source/12534/1). I'll draft this PR until more opinions are shared.

Reply:
The general question is still open, but for remake the choice is made.